statsmodels/multivariate/factor.py

```diff
@@ -0,0 +1,374 @@
+            if norm(c_last - c) < tol:
+                break
+
+        self.eigenvals = eigenvals
+        self.communality = c
+        self.loadings = A
+        return FactorResults(self)
+
+
+class FactorResults(object):
+    """
+    Factor results class
+    For result summary, scree/loading plots and factor rotations
+    Status: experimental
+
+    Parameters
+    ----------
+    factor : Factor
+        Fitted Factor class
+
+    """
+    def __init__(self, factor):
+        if not isinstance(factor, Factor):
+            raise ValueError('Input must be a `Factor` class. Got %s instead'
+                             % (factor.__str__))
+        self.endog_names = factor.endog_names
+        self.loadings_no_rot = factor.loadings
+        self.loadings = factor.loadings
+        self.eigenvals = factor.eigenvals
+        self.communality = factor.communality
+        self.rotation_method = None
+        self.fa_method = factor.method
+        self.n_comp = factor.n_comp
+
+    def __str__(self):
+        return self.summary().__str__()
+
+    def rotate(self, method):
+        """
+        Apply rotation
+
+        Parameters
+        ----------
+        method : string
+            rotation to be applied
+        -------
+        """
+        self.rotation_method = method
+        if method not in ['varimax', 'quartimax', 'biquartimax',
+                          'equamax', 'oblimin', 'parsimax', 'parsimony',
+                          'biquartimin', 'promax']:
+            raise ValueError('Unknown rotation method %s' % (method))
+
+        if method in ['varimax', 'quartimax', 'biquartimax', 'equamax',
+                      'parsimax', 'parsimony', 'biquartimin']:
+            self.loadings, T = rotate_factors(self.loadings_no_rot, method)
+        elif method == 'oblimin':
+            self.loadings, T = rotate_factors(self.loadings_no_rot, 'quartimin')
+        elif method == 'promax':
+            self.loadings, T = promax(self.loadings_no_rot)
+
+    def summary(self):
+        summ = summary2.Summary()
+        summ.add_title('Factor analysis results')
+        loadings_no_rot = pd.DataFrame(
+            self.loadings_no_rot,
+            columns=["factor %d" % (i)
+                     for i in range(self.loadings_no_rot.shape[1])],
+            index=self.endog_names
+        )
+        eigenvals = pd.DataFrame([self.eigenvals], columns=self.endog_names,
+                                 index=[''])
+        summ.add_dict({'': 'Eigenvalues'})
+        summ.add_df(eigenvals)
+        communality = pd.DataFrame([self.communality],
+                                   columns=self.endog_names, index=[''])
+        summ.add_dict({'': ''})
+        summ.add_dict({'': 'Communality'})
+        summ.add_df(communality)
+        summ.add_dict({'': ''})
+        summ.add_dict({'': 'Pre-rotated loadings'})
+        summ.add_df(loadings_no_rot)
+        summ.add_dict({'': ''})
+        if self.rotation is not None:
```

Comment:
BUG from renamings
self.rotation -> self.rotation method
also on line 318

summary() needs a smoke test.
If the result is stable and doesn't change on repeated runs, then the full summary string could be tested as equal to a saved summary.